load_test/lib/load_test/user/publisher.ex

```diff
@@ -38,7 +38,7 @@ defmodule LoadTest.User.Publisher do
       start_time: start_time
     }
 
-    Logger.info(fn ->
+    Logger.debug(fn ->
```

Comment:
Note: seems not to be required to defer the generation of the log in a function to improve performances.